### ISSUE TITLE
Deprecated Atom APIs will be removed June 1st

### ISSUE THREAD

**Issue by benogle** [lackac/terminal-runner#9]
Atom will no longer load this package after June 1st without changes. There are a couple deprecations on the latest version (`0.0.2`) of this package:

1. Store package style sheets in the `styles/` directory instead of `stylesheets/` in the `<pack-name>` package
1. Use `activationCommands` instead of `activationEvents` in your package.json Commands should be grouped by selector as follows: ```json "activationCommands": { "atom-workspace": ["foo:bar", "foo:baz"], "atom-text-editor": ["foo:quux"] } ```
1. atom.workspaceView is no longer available. In most cases you will not need the view. See the Workspace docs for alternatives: https://atom.io/docs/api/latest/Workspace. If you do need the view, please use `atom.views.getView(atom.workspace)`, which returns an HTMLElement.
1. Requiring `EditorView` from `atom` is no longer supported. Please require `TextEditorView` from `atom-space-pen-view` instead: `{TextEditorView} = require 'atom-space-pen-views'` Add `"atom-space-pen-views": "^2.0.3"` to your package dependencies.
1. Requiring `View` from `atom` is no longer supported. Please require `atom-space-pen-views` instead: `{View} = require 'atom-space-pen-views'` Add `"atom-space-pen-views": "^2.0.3"` to your package dependencies.
1. Use TextEditor::setPlaceholderText instead. eg. editorView.getModel().setPlaceholderText(text)
1. TextEditor::on is deprecated. Use documented event subscription methods instead.
1. Use ::getActivePane() instead of the ::activePane property
1. Use ::getActivePaneItem() instead of the ::activePaneItem property
1. Use ::getPaths instead

Visit https://gist.github.com/benogle/6d09e295c84b717ef9b4 and search for your package name to see up-to-date deprecations.

If this package has been replaced by another package or functionality in core, please reply with this information.

See https://github.com/atom/atom/issues/6867 for more info. Thanks!

**Comment by lackac** [comment]
Thanks for the heads up. I haven't been using Atom for a while, which explains why this package hasn't been maintained well enough.

Checking the [terminal](https://github.com/atom/terminal) package, which is a dependency of terminal-runner, looks like it hasn't been maintained either. Last commit was over a year ago.

Is there a way to deprecate or "unpublish" a package on apm? In case there is, I think that should be the fate or terminal-runner.

**Comment by benogle** [comment]
Yeah looks like we havent kept it up to date. Sorry about that. You can run `apm unpublish` when in the package repo directory.
